### PR TITLE
Load data

### PR DESCRIPTION
**Pull request by xzdandy**
- Replace old dataframe API with new storage engine.
- Integration test for load.
- Remove Loaders and uses readers.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    -   id: autopep8
+        args: ['-i', '--select=E,F', '--exclude=src/filters, src/parser/evaql']
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.8.3
+    hooks:
+    -   id: flake8
+        additional_dependencies: [flake8-typing-imports==1.6.0]
+        args: ['--select=E,F', '--exclude=src/filters, src/parser/evaql']
+
+# -   repo: https://github.com/asottile/reorder_python_imports
```
Comment (jarulraj) [inline .pre-commit-config.yaml:20]: Can we remove this?

File: README.md [hunk 2]
```diff
@@ -34,27 +34,25 @@ git clone https://github.com/georgia-tech-db/eva.git
 2. Install the dependencies.
```
Comment (jarulraj) [inline README.md:34]: Are these commands up to date?

File: script/test/test.sh [hunk 3]
```diff
@@ -1,8 +1,14 @@
 #!/bin/sh
 
 # Run linter (checks code style)
-pycodestyle  --select E src/ test/ --exclude src/filters,src/parser/evaql
-
+flake8 --select E,F src/ test/ --exclude src/filters,src/parser/evaql
+exit_code=$?
 # Run unit tests
 PYTHONPATH=./ pytest test/ --cov-report= --cov-config=.coveragerc --cov=src/ -s -v --log-level=10
```
Comment (jarulraj) [inline script/test/test.sh:8]: We could add a comment stating that this is important for `travis`.

File: src/executor/load_executor.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    def validate(self):
+        pass
+
+    def exec(self):
+        """
+        Read the input video using opencv and persist data
+        using storage engine
+        """
+        # Arbitrary batch size
+        batch_size = 50
```
Comment (jarulraj) [inline src/executor/load_executor.py:36]: Can we move `batch_size` to the configuration file?

File: src/executor/plan_executor.py [hunk 5]
```diff
@@ -57,8 +58,8 @@ def _build_execution_tree(self, plan: AbstractPlan) -> AbstractExecutor:
 
         if plan_node_type == PlanNodeType.SEQUENTIAL_SCAN:
             executor_node = SequentialScanExecutor(node=plan)
-        elif plan_node_type == PlanNodeType.STORAGE_PLAN:
-            executor_node = DiskStorageExecutor(node=plan)
+#        elif plan_node_type == PlanNodeType.STORAGE_PLAN:
```
Comment (jarulraj) [inline src/executor/plan_executor.py:61]: Can we remove this?

Comment (xzdandy) [reply src/executor/plan_executor.py]: Done

**Comment by xzdandy** [comment]
Close this pull request to make new one for the current master.

**Pull request closed without merging**